homeassistant/components/sensor/thermoworks_smoke.py

```diff
@@ -0,0 +1,151 @@
+"""
+Support for getting the state of a Thermoworks Smoke Thermometer.
+
+Requires Smoke Gateway Wifi with an internet connection.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.thermoworks_smoke/
+"""
+import logging
+
+from requests import RequestException
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import TEMP_FAHRENHEIT, CONF_EMAIL, CONF_PASSWORD,\
+    CONF_MONITORED_VARIABLES, CONF_EXCLUDE
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['thermoworks_smoke==0.1.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+PROBE_1 = 'probe1'
+PROBE_2 = 'probe2'
+PROBE_1_MIN = 'probe1Min'
+PROBE_1_MAX = 'probe1Max'
+PROBE_2_MIN = 'probe2Min'
+PROBE_2_MAX = 'probe2Max'
+
+SENSOR_TYPES = {
+    PROBE_1: 'Probe 1',
+    PROBE_2: 'Probe 2',
+    PROBE_1_MIN: 'Probe 1 Min',
+    PROBE_1_MAX: 'Probe 1 Max',
+    PROBE_2_MIN: 'Probe 2 Min',
+    PROBE_2_MAX: 'Probe 2 Max',
```

Comment:
Should be "snake case".

Reply:
The sensor names correspond directly to keys on the thermoworks side. I'd rather not change this.

Reply:
changed to snake case